idb/dummy.go

```diff
@@ -232,7 +231,7 @@ type IndexerDb interface {
 
 	CommitRoundAccounting(updates RoundUpdates, round uint64, blockPtr *types.Block) (err error)
 
-	GetBlock(round uint64) (block types.Block, err error)
+	GetBlock(ctx context.Context, round uint64, options ...GetBlockOptions) (block types.Block, transactions []TxnRow, err error)
```

Comment:
This is a neat pattern, but it's inconsistent with the other functions which use parameter objects. One example where this might be important is if the indexer ever supported co-chains, we might want one of the options to be a co-chain ID.

Since the API doesn't provide a way to pass in block parameters let's just leave off options for now and always fetch the transactions.

Reply:
adding the options was your idea.. but I don't think we can *always* fetch the transactions as it will have a negative performance impact on calls that doesn't need it.

Reply:
In that case, then a parameter object instead of an int would be fine

Reply:
oh, I think that I get what you meant by parameter objects.. you meant like in `GetAccounts`.

Reply:
Yes exactly

Reply:
ok, updated.